dymos/test/test_check_partials.py

```diff
@@ -369,6 +370,7 @@ def min_time_climb_partials_gl(self):
 
         return cpd
 
+    @require_pyoptsparse(optimizer='SNOPT')
```

Comment:
We don't have SNOPT available and should be testing these against IPOPT when possible, or SLSQP as a fallback